### PR TITLE
bus_factor += 2

### PR DESCRIPTION
**Pull request by ogrisel**
I hope @lesteve will appreciate ;)

We plan to release joblib 0.15.0 soon and this will be the first joblib version to drop Python 2 support so we should make sure the python requires meta-data to be set correctly for this one.

### CODE REVIEW

**Comment by ogrisel** [comment]
@conda-forge-admin, please rerender

**Comment by lesteve** [comment]
I edited the PR title clearly you could have made sound that more impressive :wink: 

For the record I am maintainer on plenty ROS-related packages on conda-forge because on some initial work that I did related to `pinocchio` (one of the core developer Justin is a fresh hire in the lab).

I clearly feel I am more likely to be useful on joblib than on any of these ROS-related package ...

**Comment by lesteve** [comment]
> I edited the PR title clearly you could have made sound that more impressive wink

Ah but no I can not edit the PR title of course :rofl: so here is what I suggest:
"bus_factor += 100%"

**Comment by lesteve** [comment]
And maybe you could also add @tommoral and @pierreglaser who clearly have been more active than me on joblib recently.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -35,3 +35,5 @@ extra:
   recipe-maintainers:
     - ocefpaf
     - jakirkham
+    - ogrisel
+    - lesteve
```
Comment (ogrisel) [inline recipe/meta.yaml:39]:
```suggestion
    - lesteve
    - tomMoral
    - pierreglaser
```

**Comment by ogrisel** [comment]
@conda-forge-admin, please rerender

**Comment by ogrisel** [comment]
Thanks.

**Pull request merged**